client/src/partials/stock/entry/entry.html

```diff
@@ -0,0 +1,145 @@
+<!-- fix style -->
+<style>
```

Comment:
They define the style of source buttons. We can move them into a specific style file